libensemble/tests/scaling_tests/forces/forces_gpu/run_libe_forces.py

```diff
@@ -24,11 +24,12 @@
 
 # Fixed resources (one resource set per worker)
 from libensemble.gen_funcs.sampling import uniform_random_sample as gen_f
+from libensemble.libE import libE
```

Comment:
I don't really get this change. The gen line for var resources was next to the other one.

Reply:
This is what `isort` prefers.